recipes/pcapplusplus/all/conanfile.py

```diff
@@ -13,20 +15,24 @@ class PcapplusplusConan(ConanFile):
     homepage = "https://github.com/seladb/PcapPlusPlus"
     settings = "os", "compiler", "build_type", "arch"
     options = {
+        "fPIC": [True, False],
         "immediate_mode": [True, False],
     }
     default_options = {
+        "fPIC": True,
         "immediate_mode": False,
     }
+
+    exports_sources = "patches/*"
     generators = "make"
 
-    _source_subfolder = "PcapPlusPlus"
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
```

Comment:
I'm not sure I understand this 🤔

Reply:
It's a pattern that is used in all recipes.
Granted, it's weird to name the folder `source_subfolder`

Reply:
so the pattern is calling the source directory `source_subfolder` instead of `PcapPlusPlus` (in this case)?

Reply:
Yes.